### ISSUE TITLE
Sample data for testing

### ISSUE THREAD

**Issue by maojn** [deeptools/deepTools#1015]
**Welcome to deepTools GitHub repository! Before opening the issue please check
that the following requirements are met :**

 - [x ] Search whether this issue (or a similar issue) has been solved before
 using the search tab above. Link the previous issue if appropriate below.

 - [x ] Paste your deepTools version (`deeptools --version`) and your python
 version (`python --version`) below.
deep tools 3.5.0
python 2.7
 - [ ] Paste the full deepTools command that produces the issue below
 (ignore if you simply spotted the issue in the code/documentation).

 - [ ] Paste the output printed on screen from the command that produces the issue
 below (ignore if you simply spotted the issue in the code/documentation).

Hi, Thank you for offering the software package to scientific community. I would like to test my installation with sample data using default commands. especially the matrix file for plotHeatmap. Could you tell me where I can download the detailed commands and corresponding sample data please?

**Comment by dpryan79** [comment]
You might be able to run `nosetests --with-doctest -sv deeptools`, presuming you have `nose` installed. That's done as part of our automated testing and generates some plots that you won't see and tests them for integrity. Otherwise there's really no need to test your installation.

**Comment by WardDeb** [comment]
Agreed that there is no need, but a benchmark (in context of 4.0.0) and public example data can be downloaded via the 25 repo (https://github.com/deeptools/deeptools25) or directly via zenodo (https://zenodo.org/records/21529137)
